### PR TITLE
In algob deploy/run show line number when handling script errors

### PR DESCRIPTION
**Pull request by ratik21**
previous:
image
updated to:
image

Also updated `Bad Request` error.
eg. 
image
updated to
image

### CODE REVIEW

**Review by amityadav0 — COMMENTED** [review]
Please update message to be more clear.
Instead of `:9:46` it can be `Line: 9..` or something else.

**Comment by ratik21** [comment]
> Instead of :9:46 it can be Line: 9.. or something else

`:line:charAtline` is how error stack trace (in general) shows line number in an error, which should be clear enough. But will update if you think `error at Line..` is more clear.

**Review by man-of-code — COMMENTED** [review]
I think we should remove `Algorand` name from variables and functions too. We can have this function as `parseBuilderError`. What do you think? @robert-zaremba

**Comment by robert-zaremba** [comment]
> I think we should remove `Algorand` name from variables and functions too. We can have this function as `parseBuilderError`. What do you think? @robert-zaremba

What kind of variables? We can use it in a code. Algorand is a name of product we use. We can't create other product with that name.

File: packages/algob/src/internal/util/scripts-runner.ts [hunk 1]
```diff
@@ -18,17 +18,32 @@ async function loadScript (relativeScriptPath: string): Promise<any> {
   }
 }
 
+// returns error line number with path. eg: scripts/2-gold-asc.js => scripts/2-gold-asc.js:11:3
```
Comment (amityadav0) [inline packages/algob/src/internal/util/scripts-runner.ts:21]: Please add doc string for passing variables

**Review by amityadav0 — APPROVED** [review]

File: packages/algob/src/internal/util/scripts-runner.ts (hunk 1 above)
Comment (ratik21) [reply packages/algob/src/internal/util/scripts-runner.ts:21]: ok

**Pull request merged**